### DISCUSSION TITLE
Any getting started notebook?

### DISCUSSION THREAD

**Discussion by RajS999** [rystrauss/interact#9] [Q&A]
Is there any [notebook](https://github.com/araffin/rl-tutorial-jnrr19/tree/sb3) / code snippet giving minimal example, something like [this](https://stable-baselines3.readthedocs.io/en/master/guide/quickstart.html) for stable-baselines3:

```
import gym

from stable_baselines3 import A2C

env = gym.make('CartPole-v1')

model = A2C('MlpPolicy', env, verbose=1)
model.learn(total_timesteps=10000)

obs = env.reset()
for i in range(1000):
    action, _state = model.predict(obs, deterministic=True)
    obs, reward, done, info = env.step(action)
    env.render()
    if done:
      obs = env.reset()
```

Is such usage of `interact` library possible?

Or I should go through `interact.train.py` to figure out how can I use this library in code, instead of calling it from command line as explained in readme  page.

**Comment by rystrauss** [comment]
There are no example notebooks, but `train.py` and `play.py` both contain most of what you'd need to know. Every agent implements the same interface (see the `Agent` class in `interact.agents.base`), so it's really just a matter of calling those methods. I really only wrote the code with the intention of using the command-line interface though, so that's why there aren't any code examples.

**Reply by RajS999** [reply]
Thanks for the reply. Just one more question, have you benchmarked / evaluated these algorithms? May be for performance or correctness? May be am asking for too much... 😶

**Reply by rystrauss** [reply]
I have not formally and thoroughly benchmarked them. I tested them all on one or two environments to make sure they were in fact learning, but I can't guarantee there aren't any subtle bugs that would would keep these implementations from matching published results. These implementations are mostly for my own edification and are not intended to be a well-vetted "product" like Stable Baselines or RLlib.
